lib/custom_wizard/extensions/topic_extension.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+DiscourseEvent.on(:before_create_topic) do |topic_params, user|
```

Comment:
@jumagura It's better to put this directly in the plugin.rb and without the `DiscourseEvent` prefix, i.e. to use the server-side plugin api available in the `plugin.rb`.

Reply:
Done!